src/runtime/pipeline/pipeline_function.cc

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Please go through all variable names in this file and fix the style. They should be in snake_case. There are many of them to fix.

Reply:
fixed.